third_party/externals.bzl

```diff
@@ -310,3 +310,11 @@ def register_sorbet_dependencies():
         sha256 = "47b61d25dd52bdaa1d571dab6705d076f05ba3d7a1bbbfed36145f8281c0403f",
         strip_prefix = "rules_foreign_cc-d74623f0ad47f4e375de81baa454eb106715a416",
     )
+
+    http_archive(
+        name = "rbs_parser",
+        url = "https://github.com/shopify/rbs/archive/8ddd49486c74aff543ce429a235512d0eb8cc573.zip",
```

Comment:
What is the difference between this Shopify fork of the ruby/rbs repo vs upstream?

Can we have a comment here about why we're using a fork if that's intentional?

Reply:
Yeah this is what I was trying to explain with https://github.com/sorbet/sorbet/pull/8470#discussion_r1920394406, I added better comment 👍